playbook.yaml

```diff
@@ -0,0 +1,92 @@
+- name: Install ToDo App on new web servers
+  hosts: webservers
+  remote_user: ec2-user
+  vars_prompt:
+    - name: TRELLO_API_KEY
+      prompt: What is your Trello API key?
+      private: false
+
+    - name: TRELLO_API_TOKEN
+      prompt: What is your Trello API token?
+      private: false
+
+    - name: TRELLO_BOARD_ID
+      prompt: What is your Trello Board ID?
+      private: false
+
+  tasks:
+    - name: Install git
+      yum:
+        name: git
+        state: present
+      become: yes
+
+    - name: Install python
+      yum:
+        name: python3
+        state: present
+      become: yes
+
+    - name: Install node
+      yum:
+        name: nodejs
+        state: present
+      become: yes
+
+    - name: Install poetry
+      shell: curl -sSL https://install.python-poetry.org | python3 -
+      args:
+        creates: /home/ec2-user/.local/bin/poetry
+
+    - name: Create app folder
+      file:
+        path: /opt/todoapp
+        state: directory
+        mode: "777"
```

Comment:
`777` is a good sign that we've taken a liberal attitude to permissions for the sake of getting something working, and forgotten to clean up after! Obviously not critical here, but even just dropping this to `0755` might be a simple improvement